src/pages/Files.vue

```diff
@@ -88,59 +95,54 @@ export default {
         <oc-sidebar-nav-item text="Exit ownCloud" icon="exit_to_app" target="login" />
     </oc-application-menu>
 
-    <oc-topbar variation="secondary">
-      <template slot="left">
-        <div class="uk-navbar-item uk-logo">
-          <oc-icon name="home"></oc-icon>
-        </div>
-        <oc-breadcrumb :items="[{text:'Home',to:'/'},{text:'Folder',to:{path:'folder'}},{text:'Subfolder'}]"></oc-breadcrumb>
-      </template>
-        <template slot="title">
-            <div class="uk-navbar-item">
-                <oc-search-bar label="Search"></oc-search-bar>
+    <oc-app-layout :rightHidden="rightHidden">
+
+      <template slot="center">
+        <section class="uk-background-muted uk-padding-small">
+          <div uk-grid class="uk-grid-small uk-flex-middle">
```

Comment:
the files top bar is no longer a navbar component? It should surly be - right?

Reply:
It requires a couple more sub-components. Grid, Grid Items and then the app-bar. These should become a part of a different PR is suppose. The Grid-component may become pretty complex.

Reply:
well - this sub-topbar is the more important component because it is reused in the tree apps we have as of now. We need to address this right in here as well. 

Otherwise this will totally break phoenix apart.